docs/who-is-using.md

```diff
@@ -1,18 +1,18 @@
 ## Who Is Using the Kubernetes Operator for Apache Spark?
 
-| Organization | Contact (Github User Name) | Environment | Description of Use |
+| Organization | Contact (GitHub User Name) | Environment | Description of Use |
 | ------------- | ------------- | ------------- | ------------- |
 | [Caicloud](https://intl.caicloud.io/) | @gaocegege | Production | Cloud-Native AI Platform |
-| Microsoft (MileIQ) |@dharmeshkakadia| Production | AI & Analytics |
-| Lightbend |@yuchaoran2011| Production | Data Infrastructure & Operations |
+| Microsoft (MileIQ) | @dharmeshkakadia | Production | AI & Analytics |
```

Comment:
The rest of these are whitespace. The file will not be checked by my tool going forward.